docs/getting_started_windows_cmake.md

```diff
@@ -84,14 +84,16 @@ Translate a
 and execute a function in the compiled module:
 
 ```shell
-$ .\build\iree\tools\Debug\iree-run-mlir.exe %cd%/iree/tools/test/simple.mlir -input-value="i32=-2" -iree-hal-target-backends=vmla -print-mlir
+$ .\build\iree\tools\Debug\iree-run-mlir.exe .\iree\tools\test\simple.mlir -input-value="i32=-2" -iree-hal-target-backends=vmla -print-mlir
 ```
 
 ### Further Reading
 
+*   To target GPUs using Vulkan, see
+[Getting Started on Windows with Vulkan](./getting_started_windows_vulkan.md)
+
 More documentation coming soon...
 
-<!-- TODO(scotttodd): Vulkan / other driver configuration -->
```

Comment:
Should we add a note that the beginning of the doc saying something like "this builds the core part of IREE (including ...). You will need more dependencies if to enable components like ..., which are documented in Further Reading." ?